walkthroughs/howto-k8s-http2/README.md

```diff
@@ -6,40 +6,48 @@ This example shows how to manage HTTP/2 routes in App Mesh using Kubernetes depl
 
 2. v1beta2 example manifest requires [aws-app-mesh-controller-for-k8s](https://github.com/aws/aws-app-mesh-controller-for-k8s) version [>=v1.0.0](https://github.com/aws/aws-app-mesh-controller-for-k8s/releases/tag/v1.0.0). Run the following to check the version of controller you are running.
 ```
-$ kubectl get deployment -n appmesh-system appmesh-controller -o json | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'|tail -n1
+kubectl get deployment -n appmesh-system appmesh-controller -o json | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'|tail -n1
 ```
 
 You can use v1beta1 example manifest with [aws-app-mesh-controller-for-k8s](https://github.com/aws/aws-app-mesh-controller-for-k8s) version [=v0.3.0](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/legacy-controller/CHANGELOG.md)
 
 3. Install Docker. It is needed to build the demo application images.
 
-```
+
 ## Setup
 
-1. Clone this repository and navigate to the walkthrough/howto-k8s-http2 folder, all commands will be ran from this location
+Clone this repository and navigate to the walkthrough/howto-k8s-http2 folder, all commands will be ran from this location
 1. **Your** account id:
     ```
     export AWS_ACCOUNT_ID=<your_account_id>
     ```
-1. **Region** e.g. us-west-2
+   
+2. **Region** e.g. us-west-2
```

Comment:
These numbering format with `1.` were intentional. Markdown knows how to handle it and the benefit is  when we add new items in the middle of the list we'd not want to touch other items.

Reply:
Thanks I didn't know that! Very useful indeed.